samples/subsys/audio/sof/boards/intel_adsp_cavs25.conf

```diff
@@ -4,4 +4,4 @@ CONFIG_INTEL_SSP=y
 CONFIG_INTEL_ALH=y
 CONFIG_LP_MEMORY_BANKS=1
 CONFIG_HP_MEMORY_BANKS=30
-CONFIG_RIMAGE_SIGNING_SCHEMA="tgl-h"
+CONFIG_RIMAGE_SIGNING_SCHEMA="tgl"
```

Comment:
Will you please submit this commit in a separate PR so it can be merged before the 3.0 release?

Reply:
@MaureenHelm This is harmless now and we can wait until the full thing is merged post-3.0. Let me work on the board doc. Basicly we currently really only support the 2core variant (tgl-h) in Zephyr and you can run a tgl-h binary on tgl hardware, so this ok.